React/Views/RCTModalHostView.m

```diff
@@ -168,7 +168,7 @@ - (BOOL)hasAnimationType
 
 - (void)setTransparent:(BOOL)transparent
 {
-  if (_transparent == transparent) {
+  if (self.isTransparent != transparent) {
```

Comment:
Weird...your change recommended earlier shouldn't have affected CI systems....but it must have messed up the internal Facebook tools (which I don't know).... Guess I was lucky 🤷‍♂️